whiskey.yml

```diff
@@ -1,37 +1,45 @@
 PublishOn:
+- prerelease
 - master
 
 Build:
-# The Version task sets the current build's version number. You should almost always
-# have one of these.
 - Version:
     Path: GitAutomation\GitAutomation.psd1
-    Build: $(WHISKEY_BUILD_NUMBER).$(WHISKEY_SCM_BRANCH).$(WHISKEY_SCM_COMMIT_ID)
+    Prerelease:
+    - prerelease: rc$(WHISKEY_BUILD_NUMBER)
 
-- MSBuild:
-    Path: Source\Git.Automation.sln
+- DotNet:
+    WorkingDirectory: Source
+    Command: build
+    Argument:
+    - --configuration=$(WHISKEY_MSBUILD_CONFIGURATION)
+    - -p:Version=$(WHISKEY_SEMVER2)
+    - --framework=net46
+    - --output=..\..\GitAutomation\bin
+
+- DotNet:
+    WorkingDirectory: Source
+    Command: publish
+    Argument:
+    - --configuration=$(WHISKEY_MSBUILD_CONFIGURATION)
+    - -p:Version=$(WHISKEY_SEMVER2)
+    - --framework=netstandard2.0
+    - --output=..\..\GitAutomation\bin

```

Comment:
The `GitAutomation` directory is adjacent to the `Source` directory, so shouldn't this be just `..\GitAutomation\bin` ?

Reply:
The path is relative to the .csproj being compiled, not the .sln file, so two directories up is correct.